pkgs/development/tools/dapper/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildGoPackage
+, lib
+, fetchFromGitHub
+, fetchpatch
+}:
+
+buildGoPackage rec {
+  name = "dapper-${version}";
+  version = "0.4.1";
+
+  goPackagePath = "github.com/rancher/dapper";
+
+  src = fetchFromGitHub {
+    owner = "rancher";
+    repo = "dapper";
+    rev = "v${version}";
+    sha256 = "03rmkmlvhmfwcln5v1rqww1kirxm0d1p58h6pj8f5fnhk9spb162";
+  };
+
+  meta = with lib; {
+    description = "Docker Build Wrapper";
+    homepage = https://github.com/rancher/dapper;
+    license = licenses.asl20;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ ngerstle ];
```

Comment:
I'm happy to give over being maintainer to more active/invested individuals

Reply:
Same applies here - if you want I can take it over maintenance wise. I also would like to move on with `k3s` after that as well. Did you start working on it by any chance?

Reply:
@kuznero I was just about to start, as you can guess- hence https://github.com/NixOS/nixpkgs/pull/61190

Reply:
I have a branch, but didn't create PR yet.